src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -2,6 +2,7 @@
 
 public class IntLinkedList {
 
+	public static final int IS_EMPTY = -1;
```

Comment:
This constant is misnamed. Your chosen name convention is valid for boolean contants.